opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -68,17 +68,34 @@
 #include <fmt/format.h>
 
 namespace {
-    Opm::Box makeGlobalGridBox(const Opm::EclipseGrid* gridPtr)
+    Opm::Box makeGlobalGridBox(const Opm::EclipseGrid* gridPtr,
+                               const std::vector<int>* actnum = nullptr,
+                               const std::unordered_map<int, int>* index = nullptr)
     {
         return Opm::Box {
             *gridPtr,
-            [gridPtr](const std::size_t global_index)
+            [gridPtr, actnum](const std::size_t global_index)
             {
-                return gridPtr->cellActive(global_index);
+                if (!actnum || actnum->empty()) {
+                    return gridPtr->cellActive(global_index);
+                }
+                if (actnum->empty()) {
+                    return true;
+                } else if (global_index >= actnum->size()) {
```

Comment:
I have had quite some surprises with actnum handling in OPM. Hence I just ask:
Can this really happen? When?

Reply:
yeah this is just defensive, not observed it ever happening and i doubt it can.